tlsfuzzer/analysis.py

```diff
@@ -526,10 +526,20 @@ def mt_process(self, sum_func, args=()):
         comb = list(combinations(list(range(len(self.class_names))), 2))
         job_size = max(len(comb) // os.cpu_count(), 1)
         with mp.Pool(self.workers) as pool:
-            pvals = list(pool.imap_unordered(
+            workers = set(pool._pool)
+            pvals = []
+
+            for i in pool.imap_unordered(
                 self._mt_process_runner,
                 zip(comb, repeat(sum_func), repeat(args)),
-                job_size))
+                job_size
+            ):
+                pvals.append(i)
+
+                workers.update(pool._pool)
+                if any(not p.is_alive() for p in workers):
+                    raise RuntimeError("One of workers killed")
```

Comment:
nit: "One of the workers was killed"? Also, maybe I would like to see more info on the Runtime Error, perhaps the exit code of the process.

Reply:
Exit code could be kind of useful, https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Process.exitcode

Reply:
exit code shouldn't matter too much, but I've updated it to print status like this:
```
RuntimeError: One of the workers was killed: <ForkProcess name='ForkPoolWorker-5:7' pid=38822 parent=38814 stopped exitcode=-SIGSEGV daemon>
```